bin/zopen-install

```diff
@@ -21,6 +21,25 @@ setupMyself()
 setupMyself
 checkWritable
 
+startGPGAgent()
+{
+  # Check if gpg-agent is running
+ if  ps -ef | grep -v grep | grep "gpg-agent" ; then #> /dev/null; then
```

Comment:
Can we avoid this check and just call `gpg-agent --daemon` since it already has logic to check if it's running?

```
gpg-agent --daemon
gpg-agent[67112018]: can't mmap pool of 32768 bytes: EDC5128I No such device. - using malloc
gpg-agent[67112018]: Please note that you don't have secure memory on this system
gpg-agent: a gpg-agent is already running - not starting a new one
```
note the last line

Reply:
` gpg-agent --daemon; echo $?` gives a rc of 2 when it's already running and an rc of 0 when it has not been running

Reply:
I have addressed to run daemon by default and once it is started , am checking to see if it is running or not.